### PR TITLE
Big ol' gun repricing and restocking

### PR DESCRIPTION
**Pull request by zombiemimic**
## About The Pull Request
You know how some guns are called "cheap" while being more expensive than some rifles? Not for long! Do you think there are guns that should be more accessable than they are? I'm here to fix that.

## Why It's Good For The Game
More accessibility in cheap guns, making gun prices make more sense

## Changelog
:cl:
add: added Colt, Spider Rose, Double Barrel, and Regulator to vendors
tweak: tweaked prices for Colt, Boltgun, Spider Rose, Martin, Molly, Clarissa, Cog, Havelock, and MK58
/:cl:

### CODE REVIEW

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -1561,14 +1566,17 @@
...
+					/obj/item/gun/projectile/mk58  = 900,
+					/obj/item/gun/projectile/mk58/wood = 900,
+					/obj/item/gun/projectile/colt = 1200,
 					/obj/item/gun/projectile/mandella = 1800,
 					/obj/item/gun/projectile/revolver/deckard = 3600,
 					/obj/item/gun/projectile/automatic/z8 = 3500,
-					/obj/item/gun/projectile/automatic/molly = 2000,
+					/obj/item/gun/projectile/automatic/molly = 1700,
 					/obj/item/gun/projectile/shotgun/pump/gladstone = 2200,
+					/obj/item/gun/projectile/shotgun/pump/regulator = 1900,
```
Comment (TheShown911) [inline code/game/machinery/vending.dm:1579]: The regulator is the strongest shotgun in the game, shouldn't be this cheap.

**Review by TheShown911 — COMMENTED** [review]
Have you tested how these price changes impact gun spawning?

File: code/game/machinery/vending.dm (hunk 1 above)
Comment (Jan-Blasiak) [reply code/game/machinery/vending.dm:1579]: > The regulator is the strongest shotgun in the game, shouldn't be this cheap.

Exactly

Comment (zombiemimic) [reply code/game/machinery/vending.dm:1579]: Should it be more than gladstone?

**Comment by Firefox13** [comment]
..Novakovic isn't classified as a battle rifle though?

**Comment by zombiemimic** [comment]
> ..Novakovic isn't classified as a battle rifle though?

it's the SA BR .30 "Novakovic"

**Comment by Firefox13** [comment]
Cringe, bolt-action rifles exist on their own,  might be just bolt rifle acronym.

**Comment by zombiemimic** [comment]
We need something more than the kovacs and the OS gun in the battle rifle category, that's why I moved it there

**Review by drexample — APPROVED** [review]
While I would like to make money more useful and desirable, so far to me this PR seems balanced.

**Pull request merged**